spp_area/models/area_import.py

```diff
@@ -106,10 +106,11 @@ def import_data(self):  # noqa: C901
                     for col in range(sheet.ncols):
                         col_name = sheet.cell(0, col).value
                         if col_name.find("Name") >= 0:
-                            maxcolsstr = col_name.strip("admin").replace("Name", "")
+                            maxcolsstr = col_name.strip("Area").replace("Name", "")
```

Comment:
Why change this? Do we lose compatibility with the files we used before?
We need to make sure we keep the compatibility.